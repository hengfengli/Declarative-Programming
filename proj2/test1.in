+---+---+---+
|74.|.6.|.15|
|...|..2|3..|
|9.6|58.|..4|
+---+---+---+
|.5.|..8|437|
|.9.|...|.6.|
|287|4..|.5.|
+---+---+---+
|1..|.73|5.2|
|..8|6..|...|
|37.|.9.|.81|
+---+---+---+
