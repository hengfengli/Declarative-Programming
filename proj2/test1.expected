+---+---+---+
|742|369|815|
|815|742|396|
|936|581|724|
+---+---+---+
|651|928|437|
|493|157|268|
|287|436|159|
+---+---+---+
|169|873|542|
|528|614|973|
|374|295|681|
+---+---+---+
